doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,26 +1,41 @@
 FORMAT: 1A
```

Comment:
This is weird... this PR doesn't include any modification to .apib file. Thus, why this is shown in the PR?